_projects/aimes.md

```diff
@@ -5,8 +5,21 @@ permalink: /projects/aimes
 abstract: "Enabling extreme-scale distributed computing via dynamic federation
 of heterogeneous computing infrastructures." 
 status: inactive
+logo: aimes_logo.png
 ---
 
-AIMES was a DOE ASCR funded collaborative project among the RADICAL group at Rutgers, University of Minnesota, and the Computation Institute at the University of Chicago. AIMES explored the role of abstractions and integrated middleware to support science at extreme scales. AIMES co-designed middleware from an application and infrastructure perspective, providing abstractions for compute, data and network. These were integrated across multiple levels to provide an interoperable, extensible and scalable middleware stack to support extreme-scale science. AIMES substantially contributed to the design of RADICAL-Pilot, clarifying the properties of the pilot paradigm and its architectural pattern. Further, AIMES laid the foundations for an abstraction-based study of managing the distributed execution of multi-task applications on heterogeneous and dynamic resources.
+<div class="project-description">
+	<figure>
+		<img src="../images/projects/aimes_logo.png" width="250">
```

Comment:
Here the user will not need to add the exact name. The page can pick it up from the header
```suggestion
		<img src="{{site.baseurl}}/images/projects/{{page.logo}}" width="250">
```

Reply:
We could also do something like:
```suggestion
{% if page.logo %}
  <img src="{{site.baseurl}}/images/projects/{{page.logo}}" width="250">
   {% else %}
   <img src="{{site.baseurl}}/images/projects/mgray_800x600.png" width="250">
   {% endif %}
```